WinUIGallery/WinUIGallery.DesktopWap.csproj

```diff
@@ -28,6 +28,7 @@
        in the WinUI repo, or the next ItemGroup below when standalone. -->
   <ItemGroup>
     <PackageReference Include="ColorCode.Core" />
+    <PackageReference Include="CommunityToolkit.WinUI.UI" Version="7.1.2" />
```

Comment:
If we are currently only using this for converters, it's worth checking if we can remove converters we implemented ourselVes (though I would probably just copied the converters from the toolkit or build them ourselves).